### DISCUSSION TITLE
Support for gain and offset settings

### DISCUSSION THREAD

**Discussion by muendlein** [mworion/MountWizzard4#123] [Closed Feature requests] [answered]
As of right now it seems like only the exposure time can be set. Are there any plans to allow gain and offset adjustments?

**Comment by mworion** [comment, marked as answer]
Actually not on the list, but I tag it as feature request

**Reply by mworion** [reply]
Actually in v2 beta latest, offset and gain are amplemented

**Reply by mworion** [reply]
realized
